mne/utils.py

```diff
@@ -2089,7 +2089,9 @@ def _check_fname(fname, overwrite=False, must_exist=False):
     if must_exist and not op.isfile(fname):
         raise IOError('File "%s" does not exist' % fname)
     if op.isfile(fname):
-        if not overwrite:
+        if overwrite == 'read':
```

Comment:
Added this to help reading. I was getting 'Overwriting existing file.' every time trying to read a dipole.